src/components/budget/handlers/sync-budget-records-to-funding-partners.handler.ts

```diff
@@ -101,13 +101,10 @@ export class SyncBudgetRecordsToFundingPartners
     } else {
       // TODO: mous starting at midnight on Jan 1st might cause issues with UTC vs local issues here.
       // double check how year should be calculated
```

Comment:
Can you remove this TODO as well? This addresses it.

Reply:
TODONE